client/stylesheets/_modal.scss

```diff
@@ -186,7 +186,7 @@
 
 .modOfferRequestPageOneUl li h2, .modOfferRequestPageTwo li h2, .modOfferRequestPageThree h2, 
 .modAddListingDiv h2, .modFeedbackInitialDiv h2, .modReportDiv h2 {
-	color: #404040 !important;
+	color: #555 !important;
```

Comment:
Color literals like `#555` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used<br>Line should be indented with spaces, not tabs